mpi-proxy-split/mpi-wrappers/mpi_request_wrappers.cpp

```diff
@@ -231,9 +301,8 @@ USER_DEFINED_WRAPPER(int, Probe, (int) source, (int) tag,
   return retval;
 }
 
-USER_DEFINED_WRAPPER(int, Iprobe,
-                     (int) source, (int) tag, (MPI_Comm) comm, (int*) flag,
-                     (MPI_Status *) status)
+USER_DEFINED_WRAPPER(int, Iprobe, (int) source, (int) tag, (MPI_Comm) comm,
+                     (int *) flag, (MPI_Status *) status)
```

Comment:
What changed here?

Reply:
I changed the formatting to be consistent with the other functions.